blockchain/blocks/src/tipset.rs

```diff
@@ -133,6 +134,21 @@ impl Tipset {
     pub fn weight(&self) -> &BigInt {
         self.min_ticket_block().weight()
     }
+    /// Returns true if self wins according to the filecoin tie-break rule
+    /// (FIP-0023)
+    pub fn break_weight_tie(&self, other: &Tipset) -> bool {
+        // blocks are already sorted by ticket
+        for (a, b) in self.blocks().iter().zip(other.blocks().iter()) {
+            let ticket = a.ticket().as_ref().unwrap();
+            let othtkt = b.ticket().as_ref().unwrap();
```

Comment:
Let's use a name that doesn't resemble one of the Cthulhu Mythos deities. Perhaps `other_ticket`?

Reply:
Yes, good idea ;)